tests/services/analysis_starter/test_microsalt_configurator.py

```diff
@@ -80,6 +113,13 @@ def test_get_config_path_success(
                 microsalt_case.internal_id + FileExtensions.JSON,
             ).as_posix()
         )
+        # THEN the case fastq path should be used
+        assert (
```

Comment:
This solution is perfectly fine. But thinking as LL would 👀  I'm wondering that we actually don't care about the result of the fastq handler functions just that they are called. So you could just assert that `fastq_handler.get_sample_fastq_destination_dir` gets called once and `fastq_handler.get_case_fastq_path` doesn't get called. A similar approach applies to the test above. This will decouple this test from the functionality of the fastq handler. But feel free to disregard, we can discuss more in detail our testing startegies later